test/e2e/jest-runner-vscode.config.js

```diff
@@ -2,9 +2,24 @@
 
 const path = require('path');
 
+const pkg = require('../../package.json');
+
+const vscodeVersion = '1.86.1';
+const requiredVscodeVersion = pkg.engines.vscode.match(/\d+\.\d+\.\d+/)?.[0];
```

Comment:
[note] This file is used as a cache key on CI. So, we must update this file when bumping the VSCode version.

https://github.com/stylelint/vscode-stylelint/blob/987daf7794ad227819a7fbc2ad5d85c989b2f8e2/.github/workflows/testing.yml#L49